inputfiles/removedTypes.jsonc

```
@@ -250,10 +250,8 @@
             "FontFaceDescriptors": {
                 "members": {
                     "member": {
-                        "ascentOverride": null,
-                        "descentOverride": null,
-                        "lineGapOverride": null,
-                        "variationSettings": null
+                        "display": null,
+                        "variant": null
```

```suggestion
                        "variationSettings": null // Gecko-only as of 2022-10
```

`null` value removes things. The only thing should be removed is `variationSettings` that is currently only supported in Firefox.